packages/x-components/src/components/suggestions/base-suggestion.vue

```diff
@@ -75,6 +73,15 @@
     @Prop({ default: false, type: Boolean })
     protected highlightCurated!: boolean;
 
+    /**
+     * Updates facets visibility according to prop.
+     *
+     * @returns The boolean prop value.
+     */
```

Comment:
Something like this:

```suggestion
     /**
     * Indicates if there are facets available.
     *
     * @returns True if there are facets available and false otherwise.
     * @internal
     */
```